### PR TITLE
Add a powershell sample for Google Cloud Storage.

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**

### CODE REVIEW

File: storage/powershell/Pester.Tests.ps1 [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+        (.\Copy-GcsObject.ps1 gs://$env:GOOGLE_BUCKET/testdata `
+            gs://$env:GOOGLE_BUCKET/testdata2 -Recurse).Name `
+            | Join-Output | Should Be (Groom-Expected "testdata2/
+                testdata2/hello.txt
+                testdata2/a/
+                testdata2/a/b/
+                testdata2/a/b/c.txt
+                testdata2/a/empty/")
+
+        # Now that testdata2 exists, the same copy shoud copy
```
Comment (jsimonweb) [inline storage/powershell/Pester.Tests.ps1]: ~should

File: storage/powershell/Copy-GcsObject.ps1 [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+function Split-GcsPath([string] $Path) {
+    $Path = $Path.replace('\', '/')
+    if ($Path -match '^[gG][sS]://([^/]+)/(.*)') {
+        $matches[1], $matches[2]
+    }
+}
+
+##############################################################################
+#.SYNOPSIS
+# Tests whether a Cloud Storage object with the give object name exists.
```
Comment (jsimonweb) [inline storage/powershell/Copy-GcsObject.ps1]: ~with the given object name

**Comment by jsimonweb** [comment]
Suggested a couple of typo fixes otherwise LGTM.

File: storage/powershell/Pester.Tests.ps1 (hunk 1 above)
Comment (SurferJeffAtGoogle) [inline storage/powershell/Pester.Tests.ps1]: Fixed.

File: storage/powershell/Copy-GcsObject.ps1 (hunk 2 above)
Comment (SurferJeffAtGoogle) [inline storage/powershell/Copy-GcsObject.ps1]: Fixed.

**Pull request merged**